### DISCUSSION TITLE
Does Magick++ (C++ API) support APNG output?

### DISCUSSION THREAD

**Discussion by ilvon** [ImageMagick/ImageMagick#6478] [Help]
### ImageMagick version

7.1.1.11-1 (`mingw-w64-x86_64-imagemagick`)

### Operating system, version and so on

Windows10 22H2

### Description

I saw that ImageMagick is able to read/write animated png (apng) file, however, I am not learning how to use the C++ library, Magick++, when I tried to create apng with `writeImages()`, I got all the frames being generated (static images like: xxx-1.png, xxx-2.png etc.) instead of a one single .png (or apng) file.

Here is the code that I tried to run:
```cpp
vector<Image> frames;   //storing all the frames 
writeImages(frames.begin(), frames.end(), "output.png");  //just output all the frames separately (19 frames -> 19 .png files separately)
```

Does Magick++ support creating animated png file?

**Comment by fmw42** [comment]
preface the output with APNG: such as APNG:output.png

**Reply by ilvon** [reply]
do you mean that passing output name as "APNG:output.png" to `writeImages()` for the `const std::string &imageSpec_` argument?
I have tried it but I got an error message as : "_No such file or directory @ error/blob.c/OpenBlob/3569_" when executing the code.

**Comment by ilvon** [comment]
To solve this problem, just delete the "_.../Wondershare_" folder
